homeassistant/components/xiaomi_miio/number.py

```diff
@@ -91,11 +94,17 @@
 
 
 @dataclass
-class XiaomiMiioNumberDescription(NumberEntityDescription):
+class XiaomiMiioNumberMixin:
+    """A class that describes number entities."""
+
+    method: str
+
+
+@dataclass
+class XiaomiMiioNumberDescription(NumberEntityDescription, XiaomiMiioNumberMixin):
```

Comment:
method is always defined, and seems to be compulsory so I have moved it to a separate required keys mixin